WordPress/src/main/res/layout/person_detail_fragment.xml

```diff
@@ -1,97 +1,63 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
     SlidingRelativeLayout is used for animating the slide in-out between fragments
-    Background color is set so the fragment below it, PeopleListFragment won't be visible
+    Background color is set so the fragment below it, PeopleListFragment, won't be visible
 -->
 <org.wordpress.android.ui.SlidingRelativeLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/default_background"
-    android:orientation="vertical">
+    android:background="@color/white"
+    android:clickable="true">
 
-    <RelativeLayout
+    <include
         android:id="@+id/person_details"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content">
+        layout="@layout/person_detail"/>
 
-        <org.wordpress.android.widgets.WPNetworkImageView
-            android:id="@+id/person_avatar"
-            android:layout_width="@dimen/avatar_sz_large"
-            android:layout_height="@dimen/avatar_sz_large"
-            android:layout_marginLeft="@dimen/margin_extra_large"
-            android:layout_marginTop="@dimen/margin_extra_large" />
+    <LinearLayout
```

Comment:
Can I see a visual representation of your concern?

The material guidelines suggest using the list component to open a dialog with select menu options instead of the older 'dropdown' menu pattern. In the list view we are showing the smaller role 'badge'. 

In Calypso, if you don't have permission you won't see that dropdown either.